backend/remote/remote.go

```diff
@@ -37,11 +38,12 @@ type Factory struct {
 
 type Remote struct {
 	types.ReaderWriterAt
-	name       string
-	pingURL    string
-	replicaURL string
-	httpClient *http.Client
-	closeChan  chan struct{}
+	name            string
+	pingURL         string
+	replicaURL      string
+	replicaDataAddr string // used as operation traget ID in journal
```

Comment:
Don't need the comment here. The usage of variable can be easily checked. Also, there is a typo in the comment...